File: One-Umbrella.Domain/Entities/Restaurant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OneUmbrella.Domain.Entities
{
    public class Restaurant
    {
        public int RestaurantId { get; set; }
        public int HumanId { get; set; }
        public string RestaurantName { get; set; }
        public string RestaurantStreet { get; set; }
        public string RestaurantCity { get; set; }
        public string RestaurantPostCode { get; set; }
        public string RestaurantDescription { get; set; }
        public decimal RestaurantRating { get; set; }
    }
}

File: one-umbrella.client/src/app/shared/models/restaurant/restaurant.ts
import { onlyBlob } from "../shared/onlyBlob";

export interface Restaurant {
    restaurantId : number,
    ownerId : number,
    restaurantName : string,
    restaurantStreet : string,
    restaurantCity : string,
    restaurantPostCode : string,
    restaurantRating : number,
    restaurantDescription : string,
    images : onlyBlob[]
}